src/main/java/com/example/paradise/domain/profile/dto/ProfileResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package com.example.paradise.domain.profile.dto;
+
+import com.example.paradise.domain.profile.domain.Profile;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class ProfileResponseDto {
+
+    private String profileImage;
+    private String bio;
+    private Long follower;
+    private Long following;
+
+    public ProfileResponseDto(Profile profile) {
+        this.profileImage = profile.getProfileImage();
+        this.bio = profile.getBio();
+        this.follower = profile.getFollower();
+        this.following = profile.getFollowing();
+    }
```

Comment:
한번 정적 팩토리 메서드로 구현해보는 건 어떨까요? entity를 dto롤 변환하는 로직이라는 의미에서 `from`이라는 메서드명을 가지고 있어서 service레이어에서 사용할 때에 `return ProfileResponseDto.from(profile);` 이런 식으로 명확한 의도가 담긴 코드를 작성할 수 있어요! 아래에 코드 첨부합니다!
```suggestion
    public static ProfileResponseDto from(Profile profile) {
        return new ProfileResponseDto(
            profile.getProfileImage(),
            profile.getBio(),
            profile.getFollower(),
            profile.getFollowing()
        );
    }
```
여기에서 더 추천드리는것은 Builder패턴을 같이 활용하는 방식도 있기는 한데 찾아보셔도 좋을 거 같습니다.